src/Sarif.Viewer.VisualStudio/overview.md

```diff
@@ -0,0 +1 @@
+The Microsoft SARIF Viewer extension for Visual Studio provides a convenient UI for analyzing static analysis log files and taking action on the items they contain. The Viewer intgrates with the Visual Studio environment, displaying a list of analysis results in the Error List and result details in a dockable tool window. Embedded target files can be opened right in the text editor with the error line highlighted. If the target file isn't embeeded in the log file, the Viewer will prompt you to open a local copy.
```

Comment:
>Error List [](start = 267, length = 10)

Should be lower case

Reply:
This is the formal name of that listview instance. It is capitalized this way in VS documentation.